c2rust-analyze/tests/filecheck/call1.rs

```diff
@@ -0,0 +1,48 @@
+// With a naive iteration order, reaching a fixpoint used to take 3 iterations.  By following a
+// postorder traversal of the callgraph, we reduce that to 2 iterations: the first iteration
+// computes all the right permissions, and the second checks that we've actually reached a
+// fixpoint.
```

Comment:
Does it make sense for these to be doc comments?

Reply:
This comment is describing reasoning for the `// CHECK` below, not a function or module.  Plus, I can't imagine anyone ever running `rustdoc` on these files.

Reply:
Okay, that's reasonable.